### PR TITLE
Refactor libraries so TypedDataBuffer is public, but hidden

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by natebosch** [comment]
Can you open separate PRs for separate concerns here? Moving stuff around _and_ disabling implicit casts is quite noisy.

**Comment by kevmoo** [comment]
@natebosch – PTAL

**Comment by kevmoo** [comment]
> Can you open separate PRs for separate concerns here? Moving stuff around _and_ disabling implicit casts is quite noisy.

ack

File: lib/src/typed_buffer.dart [hunk 1]
```diff
@@ -0,0 +1,426 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
```
Comment (natebosch) [inline lib/src/typed_buffer.dart]: 2020?

Comment (kevmoo) [reply lib/src/typed_buffer.dart]: ack

**Comment by natebosch** [comment]
Make sure to merge this into `null_safety` branch.

**Comment by kevmoo** [comment]
One more cleanup, then I'm on it...

**Pull request merged**